src/Cafe/HW/Latte/Renderer/Vulkan/TextureReadbackVk.cpp

```diff
@@ -79,6 +79,16 @@ uint32 LatteTextureReadbackInfoVk::GetImageSize(LatteTextureView* textureView)
 		// todo - if driver does not support VK_FORMAT_D24_UNORM_S8_UINT this is represented as VK_FORMAT_D32_SFLOAT_S8_UINT which is 8 bytes
 		return baseTexture->width * baseTexture->height * 4;
 	}
+	else if (textureView->format == Latte::E_GX2SURFFMT::R5_G6_B5_UNORM )
+	{
+		cemu_assert(textureFormat == VK_FORMAT_R8G8B8A8_UNORM);
```

Comment:
`Latte::E_GX2SURFFMT::R5_G6_B5_UNORM` is a bit tricky, because it can be emulated as either `VK_FORMAT_R5G6B5_UNORM_PACK16` or if that is not supported by the driver the Vulkan backend will fallback to `VK_FORMAT_R8G8B8A8_UNORM`. See the code [here](https://github.com/cemu-project/Cemu/blob/90eb2e01f405e502ed33fea330352e167c4a14db/src/Cafe/HW/Latte/Renderer/Vulkan/VulkanRenderer.cpp#L2372). The readback code needs to handle this correctly. For non-matching formats conversion is necessary. If you want to keep it simple you could ignore the more complicated case and return 0 when `textureFormat != VK_FORMAT_R5G6B5_UNORM_PACK16`